README.md

```diff
@@ -1543,6 +1545,47 @@ FLAG DESCRIPTIONS
     scripts), you can use this flag to easily get the IDs of created resources for further processing.
 ```
 
+## `mw app upload [INSTALLATION-ID]`
+
+Upload the filesystem of an app to a project
+
+```
+USAGE
+  $ mw app upload [INSTALLATION-ID] --source <value> [-q] [--ssh-user <value>] [--dry-run] [--delete]
+
+ARGUMENTS
+  INSTALLATION-ID  ID or short ID of an app installation; this argument is optional if a default app installation is set
+                   in the context
+
+FLAGS
+  -q, --quiet             suppress process output and only display a machine-readable summary.
+      --delete            delete remote files that are not present locally
+      --dry-run           do not actually upload the app installation
+      --source=<value>    (required) source directory from which to upload the app installation
```

Comment:
Why is this a flag and not an argument? I assumed that flags _should_ be optional an required input is given as argument.

Reply:
That's a valid point; however, this is built to mirror the flags of the (already existing) `app download` command -- to use an argument here, and a flag for `app download` would introduce further inconsistency. Ideally, both could (should) be changed to use arguments, but there are already [downstream integrations](https://github.com/mittwald/ddev) that rely on this interface.

In any case, I'd prefer to _not_ change this in this PR, but to give this more consideration in a follow-up issue.

Reply:
Created #394 as follow-up